app/routes.py

```diff
@@ -11,7 +12,7 @@
 
 from app.factory import CACHE_TIMEOUT, CachingDisabled, cache, db
 from app.models import User, Plan
-
+from app.util import newServerFunctionality
```

Comment:
I don't like the name of this function, can you rename it to something that means something? 

Also can we please switch to snake_case (PEP8). I know there is a mixed bag in here, but I'd like to get us over to PEP8 ASAP.